esi_leap/tests/objects/test_offer.py

```diff
@@ -146,3 +146,16 @@ def test_save(self):
                 self.context, o.uuid, updated_values)
             self.assertEqual(self.context, o._context)
             self.assertEqual(updated_at, o.updated_at)
+
+    def test_send(self):
+        offer_uuid = self.fake_offer['uuid']
+        with mock.patch.object(self.db_api, 'offer_get',
+                               autospec=True) as mock_offer_send:
+            mock_offer_send.return_value = self.fake_offer
+
+            res = offer.Offer.send(self.context, offer_uuid)
+
```

Comment:
You probably also want to mock the call to `marketplace_client.create_offer(marketplace_offer_dict)`; after all, we're just testing that the `send` method calls `create_offer`. Remember, these are unit tests and not integration tests.

Reply:
I don't fully get it. marketplace_client.create_offer(marketplace_offer_dict) is a function in offer.py. I guess it should be separate from test stuff?

Reply:
Yep - in a unit test of the `send_to_marketplace` function, we don't want to test code from elsewhere, such as `marketplace_client.create_offer`. We just want to verify that it's called as expected.